setup.py

```diff
@@ -1,15 +1,35 @@
-from cx_Freeze import setup, Executable
+from setuptools import setup, find_packages
+from io import open
+import qtsass
```

Comment:
To avoid having to import things, lets use 
https://github.com/spyder-ide/loghub/blob/master/setup.py#L21

The idea is avoid having to import the package itself when running the installer